documentation/API-Onboarding-and-Lifecycle.md

```diff
@@ -91,6 +91,64 @@ Once an API proposal is approved by the TSC and will be onboarded as a new Sandb
 
 Progress on these tasks is documented in tracking issues maintained by the API Backlog Working Group.
 
+If an API proposal is approved by the TSC but no onboarding tracker is initiated within three API Backlog WG sessions due to inactivity or missing response from the proposal owner, the proposal may be marked as *Frozen*. See the [Frozen API Proposals](#frozen-api-proposals) section for more information.
+
+
+## Frozen API Proposals
+
+To maintain an efficient and focused agenda during the API Backlog Working Group (WG) sessions, a formal mechanism has been introduced to handle inactive API proposals by assigning them a **"Frozen"** status.
+
+> **Note:** The transition to **Archived Proposal** described below refers specifically to the lifecycle of an API **proposal**, not an **API Repository**. Archived proposals are proposals that never progressed to onboarding or development. In contrast, Archived Repositories are deprecated or obsolete API repositories, as described in the *Archived Stage* section of the lifecycle.
+
+### Proposal Status Lifecycle Summary
+
+| **Status**   | **Description**                                                                 | **Transition Criteria**                                                                                     | **Next Step**                    |
+|--------------|----------------------------------------------------------------------------------|--------------------------------------------------------------------------------------------------------------|----------------------------------|
+| **Active**   | Proposal is under active discussion, development or onboarding.                 | - API owner attends WG meetings<br>- Recent GitHub activity (comments, PRs, updates)<br>- Onboarding tracker in progress | → Remains Active or → Frozen     |
+| **Frozen**   | Proposal is temporarily inactive but may be reactivated by contributors.        | - No activity in GitHub for **6+ weeks**<br>- API owner absent **3 WG meetings**<br>- Approved by TSC but **no onboarding tracker** after 3 WG sessions | → Reactivated (returns to Active) or → Archived |
```

Comment:
I have a question about the criteria "Approved by TSC but **no onboarding tracker** after 3 WG sessions":

My understanding is that the Onboarding Tracker issue will be created always after the TSC has approved an API Repository proposal. the Onboarding Tracker sometimes get stuck as the original API proposal owner does not nominate the initial code owner - and therefore no repository will be created. Is that the case you meant?

After a repository is created we need another description about triggers which will start a review for archiving an inactive/stale API Repository.